examples/07_advanced/plot_ica_neurovault.py

```diff
@@ -39,13 +39,17 @@
 print("Fetching Neurovault images; "
       "if you haven't downloaded any Neurovault data before "
       "this will take several minutes.")
-nv_data = fetch_neurovault(max_images=80, fetch_neurosynth_words=True)
+nv_data = fetch_neurovault(max_images=30, fetch_neurosynth_words=True)
 
 images = nv_data['images']
 term_weights = nv_data['word_frequencies']
 vocabulary = nv_data['vocabulary']
+if term_weights is None:
+    term_weights = np.ones((len(images), 2))
+    vocabulary = np.asarray(["Neurosynth is down", ":("])
```

Comment:
Love the emoji usage. I was somewhat worried about it as a potential source of confusion, but it looks like it's not returned to the user ?

Reply:
no, it will appear as the top terms shown on top of the IC plots

Reply:
OK, light suggestion to be explicit :point_down: 

```suggestion
    vocabulary = np.asarray(["Neurosynth is down", "Please try again later"])
```

Reply:
sounds good, I'll change that but won't push now to avoid restarting the build